website/i18n/zh-CN/docusaurus-plugin-content-docs/current/01-guides/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: 开始教程
+---
+
+这些教程旨在帮助你开始使用 Databend ：
+
+## 部署 Databend
+
+* [理解部署模式](../10-deploy/00-understanding-deployment-modes.md)
```

Comment:
Obviously, no corresponding document exists here.

To ensure a successful build, two options are available.
1. add all untranslated documents at once and ensure a successful build.
2. Alternatively, change the path in the file so that it points to the correct location.